source/nodejs/adaptivecards-ui-testapp/package.json

```diff
@@ -0,0 +1,58 @@
+{
+	"name": "adaptivecards-ui-testapp",
+	"version": "1.0.0",
+	"description": "Harness for UI testing for the Adaptive Cards rendering library",
+	"author": "AdaptiveCards",
+	"license": "MIT",
+	"homepage": "https://adaptivecards.io",
+	"keywords": [
+		"adaptivecards",
+		"adaptive",
+		"cards",
+		"microsoft",
+		"bot"
+	],
+	"main": "lib/adaptivecards-ui-testapp.js",
+	"types": "lib/adaptivecards-ui-testapp.d.ts",
+	"files": [
+		"lib",
+		"dist",
+		"src"
+	],
+	"scripts": {
+		"clean": "rimraf build lib dist",
+		"prebuild": "tsc",
+		"build": "webpack",
+		"watch": "webpack --watch",
+		"start": "webpack-dev-server --open",
+		"dts": "dts-generator --prefix adaptivecards-ui-testapp --project . --out dist/adaptivecards-ui-testapp.d.ts",
+		"lint": "eslint src/**/*.ts",
+		"iterate": "webpack --cache --mode=production --progress && npm run dts",
+		"release": "npm run build && webpack --mode=production && npm run dts"
+	},
+	"dependencies": {
+		"adaptivecards-controls": "^0.9.0-beta.1"
+	},
+	"peerDependencies": {
+		"adaptive-expressions": "^4.11.0",
+		"adaptivecards": "^2.9.0",
+		"adaptivecards-templating": "^2.1.0",
+		"monaco-editor": "^0.20.0"
+	},
```

Comment:
this project should be added as part of our `lerna` project in order to have it be part of our standard CI builds.

https://github.com/microsoft/AdaptiveCards/blob/main/source/nodejs/lerna.json

this will also make it so that the tests link directly against the various packages being built rather than as peer dependencies...

Reply:
actually, you should be alright -- forgot that we are using a glob for `adaptivecards*/**` in the `lerna.json` :)